scripts/segments_HALO-20240923a.md

```diff
@@ -0,0 +1,433 @@
+        "climb to FL430",
+       )
+seg4c = (slice("2024-09-23T12:44:10", "2024-09-23T13:08:51"),
+        ["straight_leg"],
+        "leg 2 towards circle c_mid",
+       )
+
+seg6 = (slice("2024-09-23T13:12:20", "2024-09-23T14:07:30"),
+        ["circle", "c_mid"],
+        "circle c_mid",
+        ["irregularity: deviation in roll angle between 13:21:57 - 13:31:10 due to deep convection"],
+       )
+
+seg7 = (slice("2024-09-23T14:11:01", "2024-09-23T14:26:28"),
+        ["straight_leg"],
+        "leg crossing circle c_mid",
+       )
+seg8 = (slice("2024-09-23T14:26:28", "2024-09-23T14:31:32"),
+         ["straight_leg", "ascent"],
+         "climb towards FL450"
+       )
+# fl change 14:26:29 - 14:31:29 and again 14:35:43 - 14:37:34
+# straight_leg and circle start still with FL change
+
+seg9 = (slice("2024-09-23T14:37:34", "2024-09-23T15:32:20"),
+        ["circle", "c_ec"],
+        "circle c_ec (on EC track)",
+        ["irregularitiy: late start due to FL change, on circle roughly since 14:32:30"]
+       )
+
+seg10 = (slice("2024-09-23T15:36:28", "2024-09-23T15:57:14"),
+         ["straight_leg"],
+         "leg crossing circle c_ec",
+         ["irregularity: peak in rool angle to -5.65deg between 15:42:23-15:42:36"]
+        )
+
+seg11 = (slice("2024-09-23T16:00:47", "2024-09-23T16:55:23"),
+         ["circle", "c_south"],
+         "circle south",
+         ["irregularity: 16:15:30 - 16:18:30 deviation due to deep convection in southern half",
+          "irregularity: 16:47:06 and until the segment ends deviation due to deep convection in northern half"]
+        )
+
+seg12 = (slice("2024-09-23T16:57:01", "2024-09-23T17:12:15"),
+         ["straight_leg"],
+         "leg towards EC track at ec_south",
+        )
+
+seg13 = (slice("2024-09-23T17:15:25", "2024-09-23T17:23:46"),
+         ["straight_leg", "ec_track", "ec_track_northward"],
+         "EC track northward",
+         ["irregularity: the EC track is very short due to very deep convection in the south and north but covers the meeting point"]
+        )
+
+# heavy deviations to navigate through deep concevtion towards next circle
+#seg14 = (slice("2024-09-23T17:23:47", "2024-09-23T17:54:38"),)
+
+seg14 = (slice("2024-09-23T17:54:38", "2024-09-23T17:57:27"),
+        ["straight_leg"],
+        "leg 1 towards circle c_west",
+        )
+
+seg15 = (slice("2024-09-23T17:57:27", "2024-09-23T18:03:16"),
+        ["straight_leg", "ascent"],
+        "climb to FL470")
+
+seg16 = (slice("2024-09-23T18:03:17", "2024-09-23T18:12:36"),
+        ["straight_leg"],
+        "leg 2 towards circle c_west",
+        )
+
+seg17 = (slice("2024-09-23T18:15:51", "2024-09-23T19:10:36"),
+        ["circle", "c_west"],
+        "circle west",
+        )
+
+seg18 = (slice("2024-09-23T19:12:57", "2024-09-23T19:16:45"),
+        ["straight_leg"],
+         "straight leg 1 crossing c_west"
+        )
+
+seg19 = (slice("2024-09-23T19:16:59", "2024-09-23T19:18:09"),
```

Comment:
Since the pattern continues once more to the other side, I would include this third roll angle tilt. Then people can decide for themselves if they want to use all wiggles or only two of the three and, if so, which ones.
```suggestion
seg19 = (slice("2024-09-23T19:16:59", "2024-09-23T19:18:43"),
```